library/core/src/str/mod.rs

```diff
@@ -4,7 +4,7 @@
 //!
 //! For more details, see the [`std::str`] module.
 //!
-//! [`std::str`]: self
+//! [`std::str`]: ../../std/str/index.html
```

Comment:
Why did you revert this link?

Reply:
`self` was referring to `core::str`(itself)

Reply:
Oh I see, `std::str` is _not_ a re-export of `core::str`. That's confusing. Good catch!